FourthTask/sorting/types/types.go

```diff
@@ -0,0 +1,40 @@
+//	Package contains person type and implements sort.Interface
+package types
+
+import (
+	"time"
+)
+
+var (
+	StartDate time.Time = time.Date(0, time.January, 1, 0, 0, 0, 0, time.UTC) //user will set data relatively Jesus birthday
```

Comment:
There are more important dates in go world, like "2006-01-02 15:04:05" for example. 
But usually you want to set auto zero-nil value for unset variables. 
like 
`var StartDate time.Time` is OK most of the times you need it. Any value set to some non-default value could lead to hardly debugable errors.